api/ocm/tools/transfer/transferhandler/plugin/options.go

```diff
@@ -0,0 +1,178 @@
+package plugin
+
+import (
+	"encoding/json"
+
+	"github.com/mandelsoft/goutils/errors"
+
+	"ocm.software/ocm/api/ocm/tools/transfer/transferhandler"
+	"ocm.software/ocm/api/ocm/tools/transfer/transferhandler/standard"
+	"ocm.software/ocm/api/utils/runtime"
+)
+
+func init() {
+	transferhandler.RegisterHandler(100, &TransferOptionsCreator{})
```

Comment:
```suggestion
	transferhandler.RegisterHandler(100, &Options{})
```

Reply:
you are right, the method can be removed